pkgs/development/python-modules/pydeck/default.nix

```diff
@@ -1,11 +1,27 @@
-{ lib, buildPythonPackage, fetchPypi, ipykernel, ipywidgets, pythonOlder, pytestCheckHook, pandas }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, ipykernel
+, ipywidgets
+, pythonOlder
+, pytestCheckHook
+, pandas
+, jinja2
+, numpy
+, traitlets
+}:
 
 buildPythonPackage rec {
   pname = "pydeck";
   version = "0.7.1";
 
   disabled = pythonOlder "3.7";
 
+  preBuild = ''
+    # unused
+    rm pyproject.toml
+  '';
+
```

Comment:
```suggestion
```

Why is this necessary? We shouldn't remove those files just because they are not used currently.